README.md

```diff
@@ -1,10 +1,32 @@
+<h1 align="center"> NimblePhxGenTemplate </h1>
+<p>
+  <img alt="Version" src="https://github.com/nimblehq/elixir-templates/actions/workflows/test_template.yml/badge.svg" />
+</p>
+
+> A Library for initialize new Phoenix projects
+
 ## Introduction
 
-Phoenix/Mix template for projects at [Nimble](https://nimblehq.co/).
+NimblePhxGenTemplate is a library for initializing new Phoenix projects at [Nimble](https://nimblehq.co/). It saves us a lot of time on the installation and configuration of new Phoenix projects.
+
+It supports for linting, testing, and much more, so developers can focus on core feature of the application. 
+
+## Features
+
+Supporting libraries:
+
+- [credo](https://hex.pm/packages/credo): A static code analysis tool with a focus on code consistency and teaching.
+- [dialyxir](https://hex.pm/packages/dialyxir): Mix tasks to simplify use of Dialyzer in Elixir projects.
+- [ex_machina](https://hex.pm/packages/ex_machina): A factory library by the creators of FactoryBot (née FactoryGirl).
+- [excoveralls](https://hex.pm/packages/excoveralls): Coverage report tool for Elixir with coveralls.io integration.
+- [mimic](https://hex.pm/packages/mimic): Mocks for Elixir functions.
+- [oban](https://hex.pm/packages/oban): Robust job processing, backed by modern PostgreSQL.
+- [sobelow](https://hex.pm/packages/sobelow): Security-focused static analysis for the Phoenix framework.
+- [wallaby](https://hex.pm/packages/wallaby): Concurrent feature tests for elixir.
 
 ## Prerequisites
 
-NimbleTemplate has been developed and actively tested with the below environment:
+**NimblePhxGenTemplate** has been developed and actively tested with the below environment:
```

Comment:
The library now named as `NimbleTemplate`, not `NimblePhxGenTemplate` anymore FYI @byhbt  ref https://github.com/nimblehq/elixir-templates/pull/81